lib/improver/nbhood/square_kernel.py

```diff
@@ -92,7 +92,7 @@ def __repr__(self):
                              self.re_mask)
 
     @staticmethod
-    def cumulate_array(cube):
+    def cumulate_array(cube_and_mask):
```

Comment:
Why not just cube and mask separately?

Reply:
All the other functions have been set up as Cubelists containing cube and (in the original code optionally) the mask so I'm just putting this in line with the other functions as cube and mask are both required to deal with any nan's correctly. I could split these into cube, mask everywhere but exactly what difference would it make to the functionality of the code that would make it worth the effort.

Reply:
No difference to functionality, but it would be clearer to use! If not now, it would have to be eventually